### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by chengda300**
Add skeleton test files to build internship class in general
Add full test files for find and list internship commands

### CODE REVIEW

**Comment by ish1506** [comment]
Should we merge this before or after #84 ?

**Comment by chengda300** [comment]
Probably after since test files are kind of less important compared to fixing the actual bugs

File: src/test/java/seedu/address/testutil/TypicalInternships.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            .withStatus(VALID_STATUS_BOBBY)
+            .withInterviewDate(VALID_INTERVIEW_BOBBY)
+            .build();
+
+    public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
+
+    private TypicalInternships() {} // prevents instantiation
+
+    /**
+     * Returns an {@code AddressBook} with all the typical persons.
```
Comment (ish1506) [inline src/test/java/seedu/address/testutil/TypicalInternships.java:108]:
```suggestion
     * Returns an {@code AddressBook} with all the typical internships.
```

**Comment by ish1506** [comment]
LGTM! 👍

**Pull request merged**